src/main/webapp/user_profile_script.js

```diff
@@ -60,6 +60,24 @@ function validateInfoForm(id) {
     return true;
 }
 
+async function loadMessages(keyString) {
+    console.log("Start loading");
```

Comment:
You can probably remove this console.log line, unless it was left here on purpose :)

Reply:
Ah I will remove this in the new commit.